api-reference.md

```diff
@@ -29,16 +29,52 @@ Example **Tangram JS** usage:
 
 ```
 var layer = Tangram.leafletLayer({
-    scene: { import: 'https://mapzen.com/carto/refill-style-more-labels/6/refill-style-more-labels.yaml', global: { sdk_mapzen_api_key: 'mapzen-xxxxxx' } },
```

Comment:
The attribution here confuses the matter, and doesn't follow the new convention published at mapzen.com/rights so removing it.